### DISCUSSION TITLE
Feature request: tap tab title to refresh

### DISCUSSION THREAD

**Discussion by Linecutterx** [Docile-Alligator/Infinity-For-Reddit#942] [Completed Ideas]
I use multi-Reddits a lot & often sort by new (!). It would be easier to access refresh if it were triggered by tapping the tab title, which I can bring up with a short reverse scroll.

**Comment by matrixes** [comment]
I agree, if nothing else because I can't find a way to refresh the feeds other than

1. Pull to refresh (which is really cumbersome if you've scrolled down far)
2. Actionbar -> Refresh option.

I'd really like to be able to either have a refresh button in the FAB or on a bottom tab.
Right now it's way too fiddly to refresh the feeds.

**Comment by Docile-Alligator** [comment]
You can customize the fab to act as a refresh button in settings.

**Comment by Tursko** [comment]
> You can customize the fab to act as a refresh button in settings.

Where can we do that? I know you can refresh by clicking the Infinity title, but that's not super intuitive. I'd much rather be able to hold down on the tab title to refresh/bring to top.

**Comment by alexrelis** [comment]
If you keep scrolling down, as far as I know, there is no easy way to go all the way up besides scrolling or pressing ... > Refresh. It would be nice if double-tapping one of the tabs could bring you back to the top so you can view the new posts akin to how iOS does it in all apps. I'd imagine it would be relatively easy to implement.

**Reply by Docile-Alligator** [reply]
Just long click the toolbar to go back to top.
